src/query.c

```diff
@@ -10,6 +14,13 @@
 #include "utils/lsyscache.h"
 #include "miscadmin.h"
 #include "parser/parsetree.h"
+#include "pg_config.h"
+
+/* Third argument to get_attname was introduced in [8237f27]. Not sure
```

Comment:
GitHub says this commit is present in tag REL_11_0. https://github.com/postgres/postgres/commit/8237f27

Reply:
Thanks. Updated comment.

Reply:
Does that also mean the macro can be for `>= 110000`?

Reply:
I do not understand. Number of arguments was changed once, in that commit.

Reply:
Reasonable. Updated.